mk/dev.mk

```
@@ -1,323 +1,52 @@
 KUMA_DIR ?= .
 ENVOY_VERSION = $(shell ${KUMA_DIR}/tools/envoy/version.sh)
-GINKGO_VERSION := v2.1.3
-GOLANGCI_LINT_VERSION := v1.49.0
-GOLANG_PROTOBUF_VERSION := v1.28.1
-HELM_DOCS_VERSION := 1.11.0
-KUSTOMIZE_VERSION := v4.4.1
-PROTOC_PGV_VERSION := v0.4.1
-PROTOC_VERSION := 3.20.0
-UDPA_LATEST_VERSION := main
-GOOGLEAPIS_LATEST_VERSION := master
-KUMADOC_VERSION := $(shell go list -f '{{.Version}}' -m github.com/kumahq/protoc-gen-kumadoc)
-DATAPLANE_API_LATEST_VERSION := main
-SHELLCHECK_VERSION := v0.8.0
-YQ_VERSION := v4.27.2
-ETCD_VERSION := v3.5.3
-HELM_VERSION := v3.8.2
-KUBE_LINTER_VERSION := v0.0.0-20220513142942-846f273ed465
 
-CI_KUBEBUILDER_VERSION ?= 2.3.2
-CI_KUBECTL_VERSION ?= v1.23.5
-# That's the version that comes from kubebuilder.
-CI_KUBE_APISERVER_VERSION ?= v1.16.4
+CI_TOOLS_DIR ?= ${HOME}/.kuma-dev
```

```suggestion
CI_TOOLS_DIR ?= ${HOME}/.kuma-dev
ifdef XDG_DATA_HOME
	CI_TOOLS_DIR := ${XDG_DATA_HOME}/kuma-dev
endif
```

for the linux users to avoid clutter